### PR TITLE
Task-15-Profile-Help-UI

### PR DESCRIPTION
**Pull request by slavasgod**
help cards

### CODE REVIEW

File: src/components/help/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  FaWallet,
+  FaClipboardList,
+  FaBook,
+} from 'react-icons/fa';
+
+import styles from './styles.module.scss';
+
+export const Help = () => {
+  return (
+    <section className={styles.cards_wrapper}>
```
Comment (OlegPanasyuk) [inline src/components/help/index.js:14]: I suggest using of destructing functionality of es6.

`const {className1, className2} = styles`

File: src/components/app.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
 import React from 'react';
-import Header from '../containers/header';
 
+import Header from '../containers/header';
+import { Help } from './help/index';
```
Comment (OlegPanasyuk) [inline src/components/app.js]: It's unnecessary to write such row. By default webpack search file index.js

Use `import { Help } from './help`

**Review by OlegPanasyuk — COMMENTED** [review]
Look for the comments in the review

File: src/components/help/index.js (hunk 1 above)
Comment (slavasgod) [reply src/components/help/index.js:14]: Css modules are not in camelcase, so if I try to destructure this object and use default variables, our eslint settings says to use camelcase instead.

**Pull request merged**